daemon/cli.c

```diff
@@ -716,6 +717,7 @@ static void cli_incoming_terminate(char* buffer, int len, struct callmaster* m,
 
    call_destroy(c);
    obj_put(c);
+   log_info_clear();
```

Comment:
Consider putting log_info_clear() right at the end of cli_incoming(). It's always safe to call it and this would handle any other cases of log_info refcount issues that could arise in the future. That would put it right before returning to the main loop, which is where it should go.